apps/game_backend/priv/repo/migrations/20240510172559_add_daily_rewards_fields_to_user.exs

```diff
@@ -0,0 +1,10 @@
+defmodule GameBackend.Repo.Migrations.AddDailyRewardsFieldsToUser do
+  use Ecto.Migration
+
+  def change do
+    alter(table(:users)) do
+      add(:last_daily_reward_claim, :utc_datetime)
```

Comment:
Please put `_at` at the end

Reply:
I did that first, but changed to match the other reward field. Since we agree is better to do this, I'll change it.